doc/ref_model/chapters/chapter03.md

```diff
@@ -124,24 +124,28 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 <a name="3.2.1.3"></a>
 #### 3.2.1.3 Virtual Storage
 
-A workload can request different types of storage based on data longevity: persistent or ephemeral storage.
+A workload can request storage based on data availability (persistent or ephemeral storage), different types of storage (HDD, SSD, etc.) and storage size.
 Persistent storage outlives the compute instance whereas ephemeral storage is linked to compute instance lifecycle.
 
-There are multiple storage performance requirements such as latency, IOPS and capacity. For example, a workload may require one of its storage device to provide low latency, high IOPS and very large/huge storage capacity (terabytes of data).
+There are multiple storage performance attributes, such as latency, IO/s (Input/Output Operations per second, a.k.a. IOPS), and throughput. For example, a workload may require one of its storage devices to provide low latency, high IO/s and very large/huge storage size (terabytes of data).
 Low Latency storage is for workloads which have strong constraints on the time to access the storage.
-High IOPS oriented storage is for workloads requiring lots of read/write actions.
-Capacity oriented storage is for workloads that need lots of volumetry without strong perfomance constraints.
-
-Storage resources have the following attributes:
-
-| Attribute           | Description                                                              |
-|---------------------|--------------------------------------------------------------------------|
-| `name`              | name of storage resources                                                |
-| `data availibilty`  | persistent or ephemeral                                                  |  
-| `performance`       | latency, IOPS, capacity                            |
-| `enhanced features` | replication, encryption                                                  |
-| `type`              | block, object or file                                                    |
-| `size`              | size in GB                                                       |
+High IO/s oriented storage is for workloads requiring lots of read/write actions.
+Large size storage is for workloads that need lots of volume without strong performance constraints.
+Note that approximate numeric ranges for the qualitative values used above are given in the 
+"["Storage Extensions"]"(https://cntt.readthedocs.io/en/latest/ref_model/chapters/chapter04.html#4.2.3) section.
```

Comment:
```suggestion
"["Storage Extensions"]"(chapter04.html#4.2.3) section.
```
Please use relative links to keep the links valid in multiple release branches.

Reply:
Thanks Greg0, better to reference within than external!